src/pages/Check/index.jsx

```diff
@@ -18,7 +18,8 @@ const Check = () => {
       const foundPackage = (await packages.length) > 0
       if (foundPackage) {
         const { id } = await packages[0]
-        history.push(`/delivery/${id}`)
+        sessionStorage.setItem('packageId', JSON.stringify(id))
+        history.push(`/delivery/${packageCode?.packageCode}`)
```

Comment:
What occurs if packageCode doesn't have packageCode?, I mean, if `packageCode?.packageCode` returns false. Then the user will be redirected to `/delivery/undefined`?

Reply:
no, as no package code was sent, the query didn't return nothing and foundPackage would be false